great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,130 @@
+from typing import Any, Dict, Iterable, List, Optional, Set, Union
+
+from great_expectations.core.batch import Batch, BatchRequest, RuntimeBatchRequest
+from great_expectations.rule_based_profiler.parameter_builder import (
+    MetricMultiBatchParameterBuilder,
+)
+from great_expectations.rule_based_profiler.types import (
+    Domain,
+    ParameterContainer,
+    get_parameter_value_by_fully_qualified_parameter_name,
+)
+from great_expectations.rule_based_profiler.types.parameter_container import (
+    ParameterNode,
+)
+
+
+class ValueSetMultiBatchParameterBuilder(MetricMultiBatchParameterBuilder):
+    """Build a set of unique values across all specified batches.
+
+    This parameter can be used in building the unique value_set from the
+    filtered domains for use in creating Expectations.
```

Comment:
@anthonyburdi I believe that the statement `from the filtered domains` is not correct.  All `ParameterBuilder` implementations will execute their computations for each and every `Domain` object.  For this reason, the `batch_request` (or `batch_list`) property of each `ParameterBuilder` has to be considered carefully.  Happy to discuss.  Thanks!

Reply:
Agree. Changed @alexsherstinsky! Please let me know if it is more clear and precise now.:
```python
This parameter builder can be used to build a unique value_set for each 
    of the domains specified by the DomainBuilder from all of the batches 
    specified. This value_set can be used to create Expectations.
    
    This unique value_set...
```

Reply:
@anthonyburdi Sounds good!  Thanks!